tests/templates/ptf_nn_agent.conf.dut.j2

```diff
@@ -0,0 +1,10 @@
+[program:ptf_nn_agent]
+command=/usr/bin/python /opt/ptf_nn_agent.py --device-socket 1@tcp://0.0.0.0:10900 -i 1-{{ nn_target_port }}@{{ nn_target_interface }} --set-nn-rcv-buffer=109430400 --set-iface-rcv-buffer=109430400 --set-nn-snd-buffer=109430400 --set-iface-snd-buffer=109430400
+process_name=ptf_nn_agent
+stdout_logfile=/tmp/ptf_nn_agent.out.log
+stderr_logfile=/tmp/ptf_nn_agent.err.log
+redirect_stderr=false
+autostart=true
+autorestart=true
+startsecs=1
+numprocs=1
```

Comment:
There was some back-and-forth about how we should handle it. The argument for making copies rather than symlinks was that we might want to make edits to the scripts/templates to support changes in pytest and we don't want to have to go back and update the corresponding ansible test.

Reply:
Got it